.storybook/main.js

```diff
@@ -15,4 +17,8 @@ module.exports = {
   },
   // REF https://github.com/storybookjs/storybook/issues/15336
   typescript: { reactDocgen: false },
+  webpackFinal: async (config) => {
+    // TODO Set up alias later
```

Comment:
I don't think we need aliases for the component library
Occasionally we need to import one component inside of another, but with aliases, there can be some dependency cycle errors
And it's not worth it to add aliases only for assets